### PR TITLE
planner: 20170425 -> unstable-2018-03-25

### PR DESCRIPTION
**Pull request by c0bw3b**
###### Motivation for this change

Closes #25771

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/office/planner/default.nix [hunk 1]
```diff
@@ -10,24 +10,26 @@
 , python
 }:
 
-let version = "20170425";
+let version = "20180325";
```
Comment (worldofpeace) [inline pkgs/applications/office/planner/default.nix]: Shouldn't this be `unstable-2018-03-25`?

Comment (c0bw3b) [reply pkgs/applications/office/planner/default.nix]: Yes it should.

File: pkgs/applications/office/planner/default.nix [hunk 2]
```diff
@@ -10,24 +10,26 @@
 , python
 }:
 
-let version = "20170425";
+let version = "unstable-20180325";
 
 in stdenv.mkDerivation {
   name = "planner-${version}";
 
   src = fetchgit {
```
Comment (jtojnar) [inline pkgs/applications/office/planner/default.nix]: Use `fetchFromGitLab`

Comment (c0bw3b) [reply pkgs/applications/office/planner/default.nix]: Done

File: pkgs/applications/office/planner/default.nix [hunk 3]
```diff
@@ -10,24 +10,28 @@
...
+  nativeBuildInputs = [
     pkgconfig
     intltool
     automake111x
     autoconf
     libtool
+  ];
 
+  buildInputs = with gnome2; [
     gnome-common
```
Comment (jtojnar) [inline pkgs/applications/office/planner/default.nix]: `gnome-common`, `gtk-doc` and most likely `scrollkeeper` should go to `nativeBuildInputs` too.

Comment (c0bw3b) [reply pkgs/applications/office/planner/default.nix]: Yes ok, it still builds and runs when doing what you suggests.

**Pull request merged**